ord_schema/validations_test.py

```diff
@@ -207,9 +203,8 @@ def test_reaction_recursive(self):
         # Assigning internal standard role to input should resolve the error
         message_input_istd = reaction_pb2.Reaction()
         message_input_istd.CopyFrom(message)
-        message_input_istd.inputs['dummy_input'].components[
```

Comment:
I can't imagine a human decided to use these line breaks, so just a note that it seems like we've changed what automatic styling is enforced

Reply:
Yes, we're using the "Google" styling now; I prefer how it handles e.g. dicts compared to before.

Reply:
Updated .style.yapf with these style options so we have more control over this in future.